_data/services.yml

```diff
@@ -33,3 +33,13 @@ companies:
     documentation: http://www.pandora.com/careers/#diversity
     date-updated: 2014
     status: Yes
+
+  - name: Pinterest
+    url: http://www.pinterest.com
+    twitter: pinterest
+    img: pinterest.png
+    diversitydata: No
```

Comment:
I marked this as 'no' since an infographic was provided as opposed to a stats page with more breakdown details into the diversity of the company. Was this a good idea? (I figured so).